### PR TITLE
feat: get active peer count

### PR DESCRIPTION
**Pull request by gaius-qi**
Signed-off-by: Gaius <gaius.qi@gmail.com>

## Description
- Get active peer count.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by jim3ma — CHANGES_REQUESTED** [review]

**Review by jim3ma — APPROVED** [review]
LGTM

File: manager/service/peer.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+package service
+
+import (
+	"context"
+	"strings"
+
+	"d7y.io/dragonfly/v2/manager/cache"
+)
+
+func (s *service) GetPeers(ctx context.Context) ([]string, error) {
```
Comment (bigerous) [inline manager/service/peer.go:26]: 一个manager集群下的peer的数量一般是什么量级的？如果太多的话，这里需要加个limit

Comment (gaius-qi) [reply manager/service/peer.go:26]: This interface is currently not used.

Comment (gaius-qi) [reply manager/service/peer.go:26]: Handler needs to add pagination feature, not limit.

**Pull request merged**